tests/e2e/fill-incorrect-identity-input-values.test.ts

```diff
@@ -37,90 +37,126 @@ describe("Fill incorrect identity input values", () => {
   });
 
   const localizedErrorStrings = locales.en.errors;
-
-  test("It should show error messages if Identity inputs are filled incorrectly in add identity", async () => {
-    expect.assertions(9);
-
-    try {
-      await authenticateToConnect();
-
-      await click("LOGINS");
-
-      await click("Add new email address");
-      expect(await text("Email address *").exists()).toBeTruthy();
-
-      await click("Add email");
-      expect(
-        await text(localizedErrorStrings.identityInputCantBeBlank).exists(),
-      ).toBeTruthy();
-
-      await write(
-        process.env.CONNECT_TEST_ACCOUNT_EMAIL ||
-          configVariables.connectTestAccountEmail,
-        into(textBox({ placeholder: "Enter your email" })),
-      );
-      await click("Add email");
-      expect(
-        await text(localizedErrorStrings.somethingWrong).exists(),
-      ).toBeTruthy();
-
-      await click("LOGINS");
-      await click("+ Add new phone number");
-      expect(await text("Phone number *").exists()).toBeTruthy();
-
-      await click("Add phone");
-      expect(
-        await text(localizedErrorStrings.identityInputCantBeBlank).exists(),
-      ).toBeTruthy();
-
-      await write(
-        "000000000000000000000000000000",
-        into(textBox({ placeholder: "Enter your phone number" })),
-      );
-      await click("Add phone");
-      expect(
-        await text(localizedErrorStrings.invalidPhoneNumberInput).exists(),
-      ).toBeTruthy();
-    } catch (error) {
-      await screenshot({
-        path: "./tests/e2e/screenshots/fill-incorrect-add-identity-input-values.test.png",
-      });
-
-      throw error;
-    }
+  const localizedStrings = {
+    accountOverview: locales.en["/account"],
+    identitiesOverview: locales.en["/account/logins"],
+    identityOverview: locales.en["/account/logins/[type]/[id]"],
+    newIdentity: locales.en["/account/logins/[type]/new"],
+    updateIdentity: locales.en["/account/logins/[type]/[id]/update"],
+  };
```

Comment:
😭 
What a pain to do on each test 😅

Reply:
A pain at first, but then you won't need to change the values everywhere :).